Description: Remove dead upstream links
 The cjb.net domain has been let lapse and is now a holding page.
 The winfield.demon.nl site is 404.
Author: Olly Betts <olly@survex.com>
Forwarded: not-needed
Last-Update: 2025-04-18

--- a/Docs/ReadMe
+++ b/Docs/ReadMe
@@ -87,28 +87,10 @@
 All other trademarks are acknowledged.
 
 
-Future Versions
----------------
-
-If you have any comments, bug reports or suggestions for future versions
-don't hesitate to write to me.
-New versions of the program will only be available if sufficient people
-are using this program. So let me know!
-
-
-Most recent version
--------------------
-
-Most recent version of Antiword can be found on the author's website:
-==>>  http://www.winfield.demon.nl/index.html  <<==
-==>>  http://antiword.cjb.net/  <<==
-
-
 Author
 ------
 
 The author can be reached by e-mail:
-antiword@winfield.demon.nl
-comments@antiword.cjb.net
+antiword@winfield.demon.nl (address may no longer work)
 
 But PLEASE read the FAQ before you write!!
--- a/Docs/antiword.1
+++ b/Docs/antiword.1
@@ -121,18 +121,8 @@
 Antiword is far from complete. Many features are still missing. Many images are
 not shown yet. Some of the images that are shown, are shown in the wrong place.
 PostScript output is only available in ISO 8859-1 and ISO 8859-2.
-.SH WEB SITES
-The most recent released version of Antiword is always available from:
-.br
-http://www.winfield.demon.nl/index.html
-.br
-or try
-.br
-http://antiword.cjb.net/
 .SH AUTHOR
-Adri van Os <antiword@winfield.demon.nl>
-.br
-or try <comments@antiword.cjb.net>
+Adri van Os
 .sp
 R.F. Smith <rsmith@xs4all.nl> and
 .br
